### PR TITLE
Add /give Command

### PR DESCRIPTION
**Pull request by user622628252416**
## Description
add /give command

## Testing
todo/see below

## Checklist
Things need to be done before this Pull Request can be merged.

### Blockers
- [x] Code is well-formatted and adheres to project style guidelines: `cargo fmt`
- [x] Code does not produce any clippy warnings `cargo clippy`
- [x] inform client about inventory updates
- [x] use item_registry instead of block_registry (blocked on #236)

### Discarded Ideas
- allow adding/removing components like minecraft:food (item components aren't even implemented yet so that seems to be out of scope for this PR)
- drop excess items to ground(blocked on #103)

### CODE REVIEW

**Comment by Snowiiii** [comment]
@user622628252416 is this ready for merge ?

File: pumpkin/src/entity/player.rs [hunk 1]
```diff
@@ -841,6 +846,90 @@ impl Player {
...
+            self.send_inventory_slot_update(&mut inventory, slot_index)
+                .await
+                .unwrap();
+
+            if remaining_items == 0 {
+                return;
+            }
+        }
+
+        warn!("{remaining_items} items ({}) were discarded because dropping them to the ground is not implemented", item.name);
```
Comment (Snowiiii) [inline pumpkin/src/entity/player.rs:931]: Please use `log::warn!`

**Comment by user622628252416** [comment]
@Snowiiii It is if CI passes

**Comment by Snowiiii** [comment]
LGTM.
Thanks @user622628252416

**Pull request merged**